Expensify/App
# [CP Staging] Fix missing header

### Details
We were missing the receipt header

### Fixed Issues
$ https://github.com/Expensify/App/issues/32831

### Tests
1. Tap `+ > Request money`
2. Enter amount > Next
3. Select a member > Next.
4. Select three dot menu > Add receipt
5. Verify that the header shows up


### Offline tests
N/A

### QA Steps
Same as tests

### PR Author Checklist



### Screenshots/Videos

https://github.com/Expensify/App/assets/22219519/6f4d8465-855f-459f-9173-962f05536b2d


<details>
<summary>Android: Native</summary>



</details>

<details>
<summary>Android: mWeb Chrome</summary>



</details>

<details>
<summary>iOS: Native</summary>



</details>

<details>
<summary>iOS: mWeb Safari</summary>



</details>

<details>
<summary>MacOS: Chrome / Safari</summary>



</details>

<details>
<summary>MacOS: Desktop</summary>



</details>
